spk/homeassistant/src/installer.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python3"
+PATH="${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+GROUP="users"
```

Comment:
I didn't know about the nobody group, but there is a [downloader](https://home-assistant.io/components/downloader.html) component that can download from given urls to the file system. Will that still work?